### ISSUE TITLE
sqlite feature request

### ISSUE THREAD

**Issue by smhmayboudi** [casbin-rs/sqlx-adapter#47]
Hi, it would be very nice to have an SQLite feature for new bees. In addition, with this feature, those who need to do a proof of concept become more simple and agile.

**Comment by hackerchai** [comment]
@smhmayboudi  Once the sqlx upgrade is done, we will work on it.

**Comment by hsluoyz** [comment]
Waiting for this PR: https://github.com/casbin-rs/sqlx-adapter/pull/43

**Comment by hsluoyz** [comment]
@hackerchai the PR: https://github.com/casbin-rs/sqlx-adapter/pull/43 is merged. I think we can work on it now.

**Comment by hackerchai** [comment]
> @hackerchai the PR: https://github.com/casbin-rs/sqlx-adapter/pull/43 is merged. I think we can work on it now.

@hsluoyz  I think we should wait for the tokio runtime bug fixed, so we can bump tokio into 1.0. Besides, the Github Action Mysql:8.0 bugis also not fixed yet.

**Comment by hsluoyz** [comment]
tokia is updated to v1.0.2 via this PR: https://github.com/casbin/casbin-rs/pull/230 . Will it unblock this issue?

**Comment by hackerchai** [comment]
@smhmayboudi @hsluoyz  see #55 
From version 0.4.1, `sqlx-adapter` starts to support sqlite backend.

**Comment by hsluoyz** [comment]
Thanks for the fix!
